.github/workflows/rust.yml

```
@@ -88,7 +88,7 @@ jobs:
         with:
           version: "v0.4.0"
       - name: Run cargo test
-        run: cargo test
+        run: cargo test || find -type f -iname '*.proptest-regressions' -exec cat {} +
```

I believe we don't need this anymore:

```suggestion
        run: cargo test
```

---

I believe that's correct, I'll commit your suggestion.